src/nurse/serializers.py

```diff
@@ -24,6 +26,24 @@ def get_expire_soon_prescriptions(self, obj):
         return PrescriptionSerializer(prescriptions, many=True).data
 
 
+class PrescriptionEmailSerializer(serializers.Serializer):
+    additional_info = serializers.CharField(
+        required=True,
+        max_length=1000,
+        error_messages={
+            "required": "Please provide additional information",
+            "max_length": "Additional information is too long",
+            "invalid": "Invalid additional information",
+        },
+        validators=[
+            lambda value: not bool(re.search(r'[<>"]', value)),
+        ],
+    )
+
+    class Meta:
+        fields = ("additional_info",)
```

Comment:
Didn't we want to send the date as well?

Reply:
not date for the first step